src/core/project/qgsproject.h

```diff
@@ -1772,9 +1808,20 @@ class CORE_EXPORT QgsProject : public QObject, public QgsExpressionContextGenera
     /**
      * Emitted when the CRS of the project has changed.
      *
+     * \see verticalCrsChanged()
```

Comment:
This is not emitted when only the vertical crs is changed, right? I think we could explicitly state that.

Reply:
OK, I've expanded the documentation further!